src/web/client/test/integration/schemaHelperUtil.test.ts

```diff
@@ -0,0 +1,791 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+import { expect } from "chai";
+import sinon from "sinon";
+import {
+    entityAttributesWithBase64Encoding,
+    schemaEntityName,
+    schemaKey,
+} from "../../schema/constants";
+import * as portalSchemaReader from "../../schema/portalSchemaReader";
+import {
+    getPortalSchema,
+    getEntity,
+    getAttributePath,
+    isBase64Encoded,
+    encodeAsBase64,
+    isWebFileV2,
+    getLanguageIdCodeMap,
+    getWebsiteIdToLanguageMap,
+    getwebsiteLanguageIdToPortalLanguageMap,
+} from "../../utilities/schemaHelperUtil";
+import {
+    portal_schema_V1,
+    portal_schema_V2,
+} from "../integration/portalSchemaReader.mock";
+import WebExtensionContext from "../../WebExtensionContext";
+import * as Constants from "../../common/constants";
+
+describe("schemaHelperUtil_ShouldReturnportalSchemaV2", () => {
+    afterEach(() => {
+        sinon.restore();
+    });
+    it("getPortalSchema_ShouldReturnportalSchemaV2", () => {
```

Comment:
why we are missing in the naming convention in each PRs?

Reply:
Will take care of it